Dockerfile

```diff
@@ -45,7 +45,7 @@ COPY --from=build /rails /rails
 WORKDIR /rails/mail-notify-integration
 
 # add Mail Notify to the Gemfile
-ARG MAIL_NOTIFY_BRANCH=v2
+ARG MAIL_NOTIFY_BRANCH=2.0.0
```

Comment:
When you say "we generally pass this argument", do you mean it gets passed in somewhere else, overriding this?

Reply:
Yep, when you build a docker image you can pass in arguments to change the output, it's a common pattern to declare and set a default like this, you see it a lot in this Dockerfile. `ARG` for images `ENV` for running containers.